tests/components/config/test_entity_registry.py

```diff
@@ -171,7 +172,7 @@ async def test_update_entity(hass, client):
 
     state = hass.states.get("test_domain.world")
     assert state.name == "after update"
-    assert state.attributes["icon"] == "icon:after update"
+    assert state.attributes[ATTR_ICON] == "icon:after update"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6212145)